test/test_public_bindings.py

```diff
@@ -296,8 +302,12 @@ def check_one_element(elem, modname, mod, *, is_public, is_all):
                 why_not_looks_public = ""
                 if elem_module is None:
                     why_not_looks_public = "because it does not have a `__module__` attribute"
+                # If a module is being migrated from foo.a to bar.a (that is entry {"foo": "bar"}),
+                # the module's starting package would be referred to as the new location even
+                # if there is a "from foo import a" inside the "bar.py".
+                modname = allow_dict["being_migrated"].get(modname, modname)
```

Comment:
If you already repopulate these in the `allow_dict` variable, this is not needed?